src/Doctrine/Orm/Filter/OrderFilter.php

```diff
@@ -271,4 +279,21 @@ protected function filterProperty(string $property, $direction, QueryBuilder $qu
 
         $queryBuilder->addOrderBy(\sprintf('%s.%s', $alias, $field), $direction);
     }
+
+    /**
+     * @return array<string, mixed>
+     */
+    public function getSchema(Parameter $parameter): array
+    {
+        if (str_contains($parameter->getKey(), ':property')) {
+            $properties = [];
+            foreach (array_keys($parameter->getExtraProperties()['_properties'] ?? []) as $property) {
+                $properties[] = [$property => ['type' => 'string', 'enum' => ['asc', 'desc']]];
+            }
+
+            return ['type' => 'object', 'properties' => $properties];
+        }
```

Comment:
We shouldn't need this as PropertyPlaceholderOpenApiParameterTrait alread declares a parameter per property. If this was for Validation we should move this to the ParameterValidationResourceMetadataCollectionFactory?